doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -153,16 +183,16 @@ DRGN-23231: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link
 
   **Affected platforms:** nRF5340
 
-.. rst-class:: v2-7-0
+.. rst-class:: v2-8-0 v2-7-0
 
-DRGN-22930 The SoftDevice Controller may de-reference a ``NULL`` pointer
+DRGN-22930: The SoftDevice Controller may de-reference a ``NULL`` pointer
   This can occur when using the vendor-specific HCI command Zephyr Write TX Power Level for a connection.
 
```

Comment:
Duplicated. I have added this issue DRGN-22930 in softdevice controller paragraph. 
You could remove it here.